src/tss2-esys/tss2-esys.vcxproj

```diff
@@ -251,10 +251,10 @@
     <ClCompile Include="api\Esys_Vendor_TCG_Test.c" />
```

Comment:
SYS and ESYS are structured differently.
in SYS, all TCG-Spec functions are under api/
in ESYS, all TPM-based functions are under api/

So this change screws over the structure of ESYS.
Of course we are screwed anyways with having a difference here to begin with...

Reply:
For the sake of consistency I think we should keep TCG defined Esys_* functions in api/ also.
Esys_Free was easy to move since it is in a separate .c file. It is more work for TR or context related functions so I left it there.

Reply:
I'd argue to either move all or none in one go... ;-)

Reply:
How about this now ? Prospone this until we move esys-context.c and esys-tr.c as well ?

Reply:
That's fine I can drop this one from the PR. We can move them all later.